recipes/pauda-mergeblast/merge.blast.pl

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env perl
+# xiechaos@gmail.com
```

Comment:
@bgruening , I would make the case for pauda-mergeblast being in with the main pauda. They live on separate urls, but are part of the same tool.